package.json

```
@@ -66,5 +66,24 @@
   },
   "cypress-cucumber-preprocessor": {
     "nonGlobalStepDefinitions": true
+  },
+  "description": "LinkFree Connect audiences to all of your content with just one link. It is an open-source alternative to [Linktree](https://linktr.ee/) implemented in JavaScript - It was initially created on a YouTube [live stream](https://www.youtube.com/watch?v=Jorl_vcp-Ew).",
+  "main": "generate.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/EddieHubCommunity/LinkFree.git"
+  },
+  "keywords": [
+    "javascript",
+    "linktree",
+    "socials",
+    "eddiehub",
+    "reactjs",
+    "react"
+  ],
+  "author": "EddieHubCommunity",
```

I think the name here should be **Eddie Jaoude**, but not sure yet.

```suggestion
  "author": {
    "name": "EddieHubCommunity",
    "url": "https://eddiehub.org"
  }
```

---

URL is already added below/above (forgot where it was).

---

@Panquesito7 it was linkfree.eddiehub.org not eddiehub.org